subprojects/diagnostics/src/test/groovy/org/gradle/api/tasks/diagnostics/DependencyReportTaskTest.groovy

```diff
@@ -85,4 +86,28 @@ class DependencyReportTaskTest extends AbstractProjectBuilderSpec {
         then:
         task.configurations == [bConf] as Set
     }
+
+    def "configuration to render could be specified by camelCase shortcut"() {
```

Comment:
❌ Please add a similar coverage to the `dependencyInsight` task. Maybe add it to the `DependencyInsightReportTaskIntegrationTest` class.